house/views.py

```diff
@@ -1,21 +1,29 @@
-from django.shortcuts import render
-from house.constants import HOME_PAGE_ROUTE, GLOBAL_PAGE_ROUTE
+from house.constants import HOME_PAGE_ROUTE, HOUSE_PAGE_ROUTE
+from .models import House
```

Comment:
why did you write here .models and not house.models?
and on the next line expenses.models?

Don't they need to have the same syntax?

Reply:
this is the house/views, so I'm already in the house model. 
and the expenses model is a separate model